### ISSUE TITLE
Integrate MQTT/WSS code from SPIKE #7739 with @stelios ACLK work

### ISSUE THREAD

**Issue by underhood** [netdata/netdata#7880]
The SPIKE #7880 (code in https://github.com/underhood/netdata/tree/mqtt_lws) result was successful but need to be merged into the code in a deployable way.

* The PR from @stelfrag's work on the application-layer needs to be merged first.
* The PoC code from the spike needs to be integrated in Netdata on top of this code.
* Any changes to netdata/mosquitto that result during #7740 need to be updated.
* This task needs to be coordinated with the actual deployment of the library into production.

## Expected result

When the PR for this issue is merged we should have fully functional transport later for the ACLK that wraps MQTT inside wss and allows connection to VerneMQ. The actual establishment of this link should remain gated behind a feature-flag until we are happy that the ACLK details are stable and we are ready to switch it on.

**Comment by underhood** [comment]
Work for this commit can be tracket at `underhood/netdata` branch `aclk_link_2` before it becomes an PR
